docs/admin/engines/torznab-engines.rst

```diff
@@ -0,0 +1,49 @@
+.. _torznab engines:
```

Comment:
Can you move the documentation to the module doc-string of [torznab.py](https://github.com/searxng/searxng/pull/2514/files#diff-fc78c908dfd1516d5cd500270519c98fede95969c3ecd348e1cea751d91a68cf)

Reply:
Done in [0726cc5](https://github.com/searxng/searxng/pull/2514/commits/0726cc55eeb19f204327de9516b19ceb4e19f486).
Is the formatting correct?